config/prometheus/workersPodMonitor.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Great. Can we add a section here and let users know how to monitor workers ? https://ray-project.github.io/kuberay/guidance/observability/

Reply:
I will add some info for monitoring workers in the docs later today!

Reply:
@Jeffwan I included some content in the observability documentation, added an prometheusRules example under `config/prometheus/rules` and made some small updates to the old serviceMonitor

Reply:
Thanks!